src/Tools/ExternalAccess/CompilerDeveloperSDK/GlobalUsings.cs

```diff
@@ -0,0 +1,5 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+global using LspRequestContext = Microsoft.CodeAnalysis.LanguageServer.Handler.RequestContext;
```

Comment:
❔ Does this not work in the project file?

```xml
<ItemGroup>
  <Using Include="LspRequestContext = Microsoft.CodeAnalysis.LanguageServer.Handler.RequestContext" />
</ItemGroup>
```

Reply:
It could, but I much prefer having these things in user-written source. Go-to-def actually works and we can modify it without switching to a separate file if necessary.